### PR TITLE
October 2023 news update

### PR DESCRIPTION
**Pull request by martincaddick**
october2023newsupdate.md and updated index.md files

### CODE REVIEW

**Comment by martincaddick** [comment]
@mattshawoobe @SamBH Please review

File: docs/news/october2023newsupdate.md
Comment (SamBH) [inline docs/news/october2023newsupdate.md:1]: Please can we make typos point 4 (as least exciting). Also, missing space in 'AzureAD'

**Review by SamBH — CHANGES_REQUESTED** [review]
Please see comment in new article.

File: docs/news/october2023newsupdate.md
Comment (martincaddick) [reply docs/news/october2023newsupdate.md:1]: Changed order and fixed AzureAD

**Review by SamBH — APPROVED** [review]

**Pull request merged**